microros_utils/library_builder.py

```diff
@@ -4,7 +4,6 @@
 
 from .utils import run_cmd
 from .repositories import Repository, Sources
-
```

Comment:
Why this removal?

Reply:
I'm sorry. It's probably some IDE auto-formatting.